### PR TITLE
Fixes tesla bolts lagging the server to shit.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request
The tesla no longer blows shit up, and will no longer zap the same thing 200 times in 1 tick.

## Why It's Good For The Game
aba

## Changelog
:cl:
balance: Electric bolts no longer explode shit.
fix: Electric bolts no longer lag the server as a result.
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
wholesome

**Comment by Doosteroni** [comment]
nonononno your fixing all wrong! Your not supposed to Improve the game you are supposed to REMOVE FEATURES! that is your function!

**Comment by CRITAWAKETS** [comment]
>tesla no longer explodes shit

what does the tesla even do now besides roam hallways and shock people

**Comment by Iamgoofball** [comment]
it zaps people

**Comment by JenqaDev** [comment]
> it zaps people

please i want boomboom!!1!1

**Comment by ATH1909** [comment]
Have you tested this to make sure that it actually reduces lag/doesn't break anything horribly? What if 5-10 teslas are released at once?

**Comment by Iamgoofball** [comment]
The lag was from the explosions which would then cripple atmos which then leads to the rest of the game exploding. It's fine.

**Comment by LittleNyanCat** [comment]
Now the true question is if this is gonna change CRITAWAKETS or orangemanbad's opinion on that other PR.... spoiler alert it probably won't but I very much cherish your genuine attempt at fixing something instead of outright removing it

**Comment by FloranOtten** [comment]
> Now the true question is if this is gonna change CRITAWAKETS or orangemanbad's opinion on that other PR.... spoiler alert it probably won't but I very much cherish your genuine attempt at fixing something instead of outright removing it

CORRECT! He merged it. This will now have to be Readds tesla and makes it not lag

**Comment by LittleNyanCat** [comment]
Wow, just wow.... I'm spechless.

**Comment by silicons** [comment]
based

**Comment by Hopekz** [comment]
What's the point if it doesn't actually explode? 
Kind of dumb to just have a stunning ball roam the station with no actual impact on the physical station itself.

**Comment by silicons** [comment]
depending on how much power it outputs and what the scaling is it should still erase a ton of machines, it just won't breach or anything in most cases

**Comment by wesoda25** [comment]
This is just bandaid for now I think. Better than lag 🤷‍♂️

**Comment by LemonInTheDark** [comment]
this just works. Makes the tesla a bit weak on the overall, but that can be addressed later

**Comment by LemonInTheDark** [comment]
mmmm done some more testing, atmos is still a minor concern cause they can slowly break airlocks and cause siphons, but it's so much better then it was before. Garbage collection is still an issue, but this is with like 50 teslas and the game is at 80% td. oh also the timer system is taking a hit, I assume that's the 50 teslas and the zap cooldowns

**Comment by Iamgoofball** [comment]
@Hopekz stunning? buddy tesla kills

**Comment by kopoba** [comment]
can it just woosh things like disintegrate without explosion?

File: code/__DEFINES/flags.dm [hunk 1]
```diff
@@ -127,7 +127,7 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 #define ZAP_MOB_DAMAGE			(1<<3)
 #define ZAP_MOB_STUN			(1<<4)
 
-#define ZAP_DEFAULT_FLAGS ALL
+#define ZAP_DEFAULT_FLAGS ZAP_MOB_STUN | ZAP_MOB_DAMAGE | ZAP_OBJ_DAMAGE 
 #define ZAP_FUSION_FLAGS ZAP_OBJ_DAMAGE | ZAP_MOB_DAMAGE | ZAP_MOB_STUN
```
Comment (ShizCalev) [inline code/__DEFINES/flags.dm:131]: why not just use this?

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by Iamgoofball** [comment]
this is literally ready to merge

**Comment by LemonInTheDark** [comment]
Update the changelog to reflect the tesla not being in the game anymore and this pr instead effecting it's eventual re-addition/other things that use this bitfield, and I'll merge it

**Comment by LemonInTheDark** [comment]
listen man I was working on sm code, and suddenly it was 5 hours later and I had forgotten to merge goof's pr.

**Comment by LemonInTheDark** [comment]
forgive the edit, don't wanna stir up reddit too much

**Pull request merged**